src/tests/unit/tests/electron/platform/android/appium-service-configurator-factory.test.ts

```diff
@@ -32,6 +38,7 @@ describe('AppiumServiceConfiguratorFactory tests', () => {
 
         adbCreatorMock.verifyAll();
```

Comment:
3 mocks is usually my "threshold" for moving the verifyAll calls into a method called verifyAllMocks

Reply:
done